src/utils/breaches.js

```diff
@@ -24,10 +25,25 @@ import { filterBreachDataTypes } from './breachResolution.js'
  * @param {*} allBreaches
  * @returns {Promise<AllEmailsAndBreaches>}
  */
-async function getAllEmailsAndBreaches (user, allBreaches) {
-  const monitoredEmails = await getUserEmails(user.id)
+async function getAllEmailsAndBreaches(user, allBreaches) {
+  
+  // @ts-ignore: function will be deprecated
   const verifiedEmails = []
+  // @ts-ignore: function will be deprecated
```

Comment:
Is this `@ts-ignore` comment saying the `getallEmailsAndBreaches()` function will be deprecated?
Also, curious why we need these two ignore statements (plus L37 and L42).